### PR TITLE
July 2024 What's New monthly blog post

### PR DESCRIPTION
**Pull request by sarah11918**
Monthly blog post of content, showcase etc.

### CODE REVIEW

File: src/content/blog/whats-new-july-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+import ShowcaseGridById from './_whats-new-components/ShowcaseGridById.astro';
+import ThemeGrid from './_whats-new-components/ThemeGrid.astro';
+import YouTubeGrid from './_whats-new-components/YouTubeGrid.astro';
+
+Some exciting news from around the Astro ecosystem this month. Let's dive in!
+
+## Updates from Mission Control
+
+- 🏝️ [Server Islands (experimental)](https://docs.astro.build/en/reference/configuration-reference/#experimentalserverislands)
+- 🤝 [Netlify Partnership ](https://astro.build/blog/netlify-official-deployment-partner/)
```
Comment (ascorbic) [inline src/content/blog/whats-new-july-2024.mdx:21]:
```suggestion
- 🤝 [Netlify Partnership](https://astro.build/blog/netlify-official-deployment-partner/)
```

File: src/content/blog/whats-new-july-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+- 📄 [Why we use Astro](https://www.pricelevel.com/blog/why-we-use-astro) by @rapp
+- 📄 [Astro Actions With Vanilla JavaScript and Strapi 5](https://strapi.io/blog/astro-actions-with-vanilla-javascript-and-strapi5) by @paul-bratslavsky
+- 📄 [Creating a portfolio website using Astro and Atomic Server](https://docs.atomicdata.dev/astro-guide/1-index) by @AtomicDataDocs
+- 📄 [Launching Your Astro Powered Blog on a Journey to Couchbase](https://dev.to/couchbase/launching-your-astro-powered-blog-on-a-journey-to-couchbase-3kh8) by @bengreenberg
+- 📄 [SSR Astro with Headless Craft CMS](https://www.olets.dev/posts/ssr-astro-with-headless-craft-cms) by @olets
+- 📄 [Astro vs Next.js](https://caisy.io/blog/astro-vs-next) by @Caisy
+- 📄 [Why I ditched WordPress for Astro](https://vbartalis.xyz/en/blog/why-i-ditched-wordpress-for-astro-js/) by @vbartalis
+- 📄 [Astroのrssフィードに記事の本文を追加する](https://www.neputa-note.net/2024/07/astro-rss/) by @neputa
+- 📄 [How to do ISR and advanced caching with Astro](https://developers.netlify.com/guides/how-to-do-advanced-caching-and-isr-with-astro/) by @netlify
```
Comment (ascorbic) [inline src/content/blog/whats-new-july-2024.mdx:70]: It's by me!

Comment (sarah11918) [reply src/content/blog/whats-new-july-2024.mdx:70]: Then let's get you in there!

Comment (sarah11918) [inline src/content/blog/whats-new-july-2024.mdx:70]:
```suggestion
- 📄 [How to do ISR and advanced caching with Astro](https://developers.netlify.com/guides/how-to-do-advanced-caching-and-isr-with-astro/) by @ascorbic
```

File: src/content/blog/whats-new-july-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+- 📄 [Astroのrssフィードに記事の本文を追加する](https://www.neputa-note.net/2024/07/astro-rss/) by @neputa
+- 📄 [How to do ISR and advanced caching with Astro](https://developers.netlify.com/guides/how-to-do-advanced-caching-and-isr-with-astro/) by @ascorbic
+- 📄 [Exploring actions and request rewriting in Astro](https://blog.logrocket.com/exploring-actions-request-rewriting-astro/) by @ivaylogerchev
+- 📄 [How to use Astro actions, server islands and env to show what you're currently listening on Spotify](https://hounie.me/writings/how-i-use-astro-actions-and-server-islands-spotify/) by @Guilhermo
+- 📄 [Introducing AstroAspore: using .NET Aspire with Astro SSR (Nodejs/Express)](https://agramont.net/blog/astroaspire-overview/)
+
+
+ ... and don't miss the latest tips from [Astro Tips](https://astro-tips.dev) such as [How to add GSAP to your Astro project](https://astro-tips.dev/tips/how-to-add-gsap/) and tutorials from [Lexingon Themes](https://lexingtonthemes.com/tutorials/) such as [How to a create a color picker with Tailwind CSS and JavaScript](https://lexingtonthemes.com/tutorials/how-to-create-a-color-picker-with-tailwind-css-and-javascript/).
```
Comment (delucis) [inline src/content/blog/whats-new-july-2024.mdx:76]: Small typo:
```suggestion
 ... and don't miss the latest tips from [Astro Tips](https://astro-tips.dev) such as [How to add GSAP to your Astro project](https://astro-tips.dev/tips/how-to-add-gsap/) and tutorials from [Lexington Themes](https://lexingtonthemes.com/tutorials/) such as [How to a create a color picker with Tailwind CSS and JavaScript](https://lexingtonthemes.com/tutorials/how-to-create-a-color-picker-with-tailwind-css-and-javascript/).
```

File: src/content/blog/whats-new-july-2024.mdx [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+### Tips & Tools
+
+The latest community-built utilities and integrations to help you build with Astro.
+
+- [Astro Studio CMS](https://astro-studiocms.xyz/) - A dedicated CMS for Astro Studio built by the Astro community by @AdamMatthiesen, @Jumper, and @dreyfus
+- [Advanced Astro Starter kit](https://github.com/BuckyBuck135/Advanced-Astro-v4-i18n) - Includes a pre-configured multi-language setup, and CodeStitch components by @buckybuck 
+- [Notion-based configurable static website generator](https://www.producthunt.com/posts/webtrotion) - Your own Notion website with Astro by @nerdymomocat
+- [Astro Infisical](https://www.npmjs.com/package/@matthiesenxyz/astro-infisical) - An Astro integration for Infisical for secret management by @AdamMatthiesen
+- [Bun adapter for Astro](https://github.com/nurodev/astro-bun) - Allows Astro to deploy your SSR site to Bun targets by @NeuroDev
+- [Back of Tricks - Inspection Chamber](https://github.com/martrapp/astro-vtbot) - See your view transitions like never before: examine all details, reveal, debug, and optimize! by @martrapp
```
Comment (delucis) [inline src/content/blog/whats-new-july-2024.mdx:114]:
```suggestion
- [Bag of Tricks - Inspection Chamber](https://github.com/martrapp/astro-vtbot) - See your view transitions like never before: examine all details, reveal, debug, and optimize! by @martrapp
```

**Review by delucis — COMMENTED** [review]
Spotted a couple of small typos but otherwise looks great!

File: src/content/blog/whats-new-july-2024.mdx [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+## Updates from Mission Control
+
+- 🏝️ [Server Islands (experimental)](https://docs.astro.build/en/reference/configuration-reference/#experimentalserverislands)
+- 🤝 [Netlify Partnership](https://astro.build/blog/netlify-official-deployment-partner/)
+- 🔐 [Clerk Community SDK for Astro](https://clerk.com/changelog/2024-03-28)
+- 🎟️ [Astro at ViteConf](https://viteconf-2024.pages.dev/24/ecosystem/astro)
+
+## Releases
+
+- Astro [4.12](https://astro.build/blog/astro-4120/) and Astro [4.13](https://astro.build/blog/astro-4130/)
```
Comment (sarah11918) [inline src/content/blog/whats-new-july-2024.mdx:27]:
```suggestion
- Astro [4.12](https://astro.build/blog/astro-4120/)
```
not released until tomorrow, Aug 1! (so, blog post is not live)

File: src/content/blog/whats-new-july-2024.mdx [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+
+Several new Astro themes were added to the [Astro theme directory](https://astro.build/themes/) this month. Try them out!
+
+ <ThemeGrid themes={['astro-blur', 'i18n-starter', 'astrozinc', 'kubrick-astro', 'atemp-astro', 'saaspro-astro', 'elkamali-portfolio-blog', 'astrolinkhub', 'eclipse', 'astrorante', 'stellar-astro-dev-portfolio', 'nehru', 'logbook-astro', 'void-astro', 'foxi', 'print-friendly-portfolio-cv', 'astro-micro-academics', 'base-astro', 'astro-x-shadcn-minimal-theme', 'starpod', 'starlight-pwa', 'small-business-template', 'fortify-astro']} />
+
+Also check out:
+
+- [Freedom Stack](https://github.com/cameronapak/freedom-stack) by @cameronapak
+- [ToolDB](https://protocolui.com/templates/tooldb) by @ProtocolUI
+- [Print-friendly portfolio CV](https://github.com/Smilesharks/dev-portfolio)
```
Comment (sarah11918) [inline src/content/blog/whats-new-july-2024.mdx:134]:
```suggestion
- [Print-friendly portfolio CV](https://github.com/Smilesharks/dev-portfolio) by @smilesharks
```

**Pull request merged**